### PR TITLE
Improve error message when custom cfg file does not exist

### PR DESCRIPTION
**Pull request by LegenJCdary**
fixing issue #100

### CODE REVIEW

File: source/main.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def parse_options(argv):
                         'and errors to syslog. --debug doesn\'t affect this option!')
     parser.add_argument("--limit", "-l", nargs=1, default=[None], metavar="[LIMIT]",
                         help='Limit task execution to specified host.')
-    parser.add_argument("--conf-dir", nargs=1, default=[None], metavar="conf_dir",
+    parser.add_argument("--conf-dir", "-C", nargs=1, default=[None], metavar="conf_dir",
```
Comment (cinek810) [inline source/main.py:41]: it's strange that it didn't require tens of changes in tests... maybe tets are failing and we're not aware?

File: setup.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def read(fname):
 # This call to setup() does all the work
 setup(
     name="ansible-deployer",
-    version="0.0.24",
+    version="0.0.27",
```
Comment (cinek810) [inline setup.py:22]: it's always fine, but it's also one of the things that may result in conflicts so.. sometimes is better not to touch it :)

**Review by cinek810 — CHANGES_REQUESTED** [review]

File: source/main.py (hunk 1 above)
Comment (LegenJCdary) [reply source/main.py:41]: Thankfully (to you tbh since you were adding tests for --dry and --conf-dir) we use long version for those options.

Comment (cinek810) [reply source/main.py:41]: OK.. I thought that it's possible. We need an issue to check both (I'll create one)

**Review by cinek810 — APPROVED** [review]

**Pull request merged**